wellcomeml/viz/visualize_clusters.py

```diff
@@ -0,0 +1,31 @@
+from bokeh.plotting import figure, show
+from wellcomeml.viz.palettes import Wellcome33
+from bokeh.io import output_notebook
```

Comment:
Import `output_file` too (see below for a suggestion)

Reply:
Done